### ISSUE TITLE
"must see"

### ISSUE THREAD

**Issue by ethanachi** [UniversalDependencies/UD_English-EWT#121]
An interesting border case happens here:

http://match.grew.fr/?corpus=UD_English-EWT@2.7&custom=5feef22b3ff08&eud=yes

where "must see" is pretty clearly a `compound` with an adjectival function, but the validator assumes that `AUX` (and specifically `AUX`) can never be in a compound...

My feeling is that the compound analysis is correct and the validator is making false assumptions, but curious as to others' opinions.

**Comment by nschneid** [comment]
Yeah I have noticed similar issues where a phrase with compositional structure is converted to an attributive modifier, like "make-or-break decision" and "easy-to-use tool"—should we recognize the internal structure reflecting the derivational origin of the expression, or just use `compound` across the board?

**Comment by nschneid** [comment]
Related: UniversalDependencies/docs#648, UniversalDependencies/docs#478, UniversalDependencies/docs#525

**Comment by dan-zeman** [comment]
Hmm, is it still `AUX` if it enters a compound? Isn't it `VERB` or something then?

**Comment by nschneid** [comment]
I think it's a compound in the sense that "church-going" is a compound in "he is a church-going person". But is `compound` intended for a narrower use of combining two words with like parts of speech?

**Comment by dan-zeman** [comment]
I don't think that the guidelines explicitly ban auxiliaries from the `compound` relation (although I did not check carefully now). I would expect auxiliaries to be more likely to participate in `fixed` than in `compound` (because they are function words) but _must see_ is obviously not what UD annotates as `fixed`. The reason why I made the validator report `AUX`-`compound` as an error was (if I recall it correctly) that some treebanks annotated light verbs as `AUX`, which is wrong.

Are there other compounds that contain auxiliaries? Or should we say that _must see_ is an English-specific exception?
